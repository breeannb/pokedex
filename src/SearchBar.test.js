import React from 'react'; 
import { shallow } from 'enzyme'; 
import SearchBar from './SearchBar.js';

it('renders a searchbar', () => {
  const wrapper = shallow(<SearchBar />);

  expect(wrapper).toMatchSnapshot();

  
}); 