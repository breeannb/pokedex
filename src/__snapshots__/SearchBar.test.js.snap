// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a searchbar 1`] = `ShallowWrapper {}`;
